src/backoff.ts

```diff
@@ -1,3 +1,7 @@
+export class RetryError extends Error {
+  name: 'RetryError';
```

Comment:
@louib this isn't enough to get the stringify behavior you expect, you need

```js
class RetryError extends Error {
  constructor(message) {
    super(message);
    this.name = "RetryError";
    // Or
    this.name = this.constructor.name
  }
}
```

That's what we do in https://github.com/unitoio/ucommon/blob/master/src/lib/BaseError.ts#L56-L61